### DISCUSSION TITLE
Spinner type in notification

### DISCUSSION THREAD

**Discussion by chupins** [zauberzeug/nicegui#3959] [Ideas / Feature Requests]
Hello,

I'm using a ui.notification with spinner=True argument but i'll like to change the type and size of the spinner (like it's possible in ui.spinner).

Is there any way to do this ?

**Comment by falkoschindler** [comment]
Hi @chupins,

Unfortunately it doesn't seem to be possible to change the spinner in a notification. Even though the [Quasar documentation](https://quasar.dev/quasar-plugins/notify) says you can pass a string like `spinner='QSpinnerBars'`, their [example](https://quasar.dev/quasar-plugins/notify#example--with-spinner) suggests that you need to pass an object like `spinner=QSpinnerBars` which is imported from `quasar`. Even if we add dynamic property conversion to `ui.notify`, I don't see an easy way to get this working. But let's convert this question into a feature request. Maybe someone wants to give it a try.
